packages/frontend/src/components/UnderlyingData/UnderlyingDataProvider.tsx

```diff
@@ -219,6 +219,9 @@ export const UnderlyingDataProvider: FC<Props> = ({
                           },
                       ] = r;
 
+                      if (raw === null) {
+                          return acc;
```

Comment:
I don't think is going to work when there are null and non null values in the same columns. 
If we don't apply a filter it would return rows where that column has values.

Reply:
mmm, is that possible ? This issue only affects `Null`s on dimensions, and a different value in a dimension should appear on a different row, right ? :thinking:

Reply:
But maybe a better solutionis  to add a filter with the `IS NULL` operator on it anyway

Reply:
That works 

![image](https://user-images.githubusercontent.com/1983672/187705589-37be29d0-9949-4eb0-8964-5afb1572fadd.png)